pkg/combustion/registry.go

```diff
@@ -232,6 +232,8 @@ func configureEmbeddedArtifactRegistry(ctx *image.Context) (bool, error) {
 		}
 	}
 
+	zap.S().Infof("Preparing to add the following images to the embedded artifact registry:\n%s", images)
```

Comment:
I'd move this log just before `populateRegistry()` or even within it and reword this to `Adding the following images to the embedded artifact registry:`.